### ISSUE TITLE
[Feedback] Confusion with Next.js tutorial

### ISSUE THREAD

**Issue by khristinasar** [aws-amplify/docs#2627]
**Page**: [`/start/getting-started/data-model/q/integration/next`](https://docs.amplify.aws/start/getting-started/data-model/q/integration/next)

**Feedback**:

Hi,

It looks like the `Connect API and database to the app` tutorial advises to choose schema template for a `"Todo” app`:

```
? Choose a schema template:
# Single object with fields (e.g., “Todo” with ID, name, description)
```

But then it continues about the `"Blog" app`. It's a little bit confusing. 

Then I got this error:
```
Error: Amplify has not been configured correctly. 
The configuration object is missing required auth properties. 
Did you run `amplify push` after adding auth via `amplify add auth`?
```
So I had to find another tutorial for Authentication and follow the instructions there to get rid of that error.

Note: I didn't know I had to press a `space bar` instead of `enter` for the question below to actually choose "Amazon Cognito User Pool" which caused an error when I tried to create a new Post, even though I was logged in:
`? Choose the additional authorization types you want to configure for the API Amazon Cognito User Pool`

**Comment by Khristinasheer** [comment]
Update: I was able to fix my error by doing 2 things: 

1. I followed the instructions from here: https://docs.amplify.aws/lib/auth/getting-started/q/platform/js. I'm not sure why in the Tutorial it doesn't say anything about running `amplify add auth`.

2. I added the following code that was recommended here: https://github.com/aws-amplify/amplify-js/issues/4315
```
Amplify.configure({ ...awsExports, ssr: true });
Auth.configure(awsExports);
```
